spec.html

```diff
@@ -32164,27 +32198,42 @@ <h1>Runtime Semantics: RegExpBuiltinExec ( _R_, _S_ )</h1>
             1. Assert: The value of _A_'s `"length"` property is _n_ + 1.
             1. Perform ! CreateDataProperty(_A_, `"index"`, _lastIndex_).
             1. Perform ! CreateDataProperty(_A_, `"input"`, _S_).
-            1. Let _matchedSubstr_ be the matched substring (i.e. the portion of _S_ between offset _lastIndex_ inclusive and offset _e_ exclusive).
-            1. Perform ! CreateDataProperty(_A_, `"0"`, _matchedSubstr_).
-            1. If _R_ contains any |GroupName|, then
+            1. Let _indices_ be a new empty List.
+            1. Let _match_ be the Match { [[StartIndex]]: _lastIndex_, [[EndIndex]]: _e_ }.
+            1. Add _match_ as the last element of _indices_.
+            1. Let _matchedValue_ be ! GetMatchString(_S_, _match_).
+            1. Perform ! CreateDataProperty(_A_, `"0"`, _matchedValue_).
```

Comment:
Is this to align with https://github.com/tc39/ecma262/pull/1676?

Reply:
I've reverted this line (and changing the name of the variable on the line above to match), since it is likely already addressed by #1676 and will be one less merge conflict.